questions/questions.py

```diff
@@ -1,102 +1,166 @@
+        responses = {}
+        q_message = cast(discord.Message, DummyMessage(copy.copy(initial_message)))
         q_message.author = self.bot.modmail_guild.me
 
+        # Get the timeout value from config or use the default of 15 minutes
+        timeout = int(config.get('timeout', 15 * 60))  # default to 15 minutes (900 seconds)
+
         for question in config['questions']:
-            print(question)
             q_message.content = question
             await thread.reply(q_message)
 
-            try:
-                m = await self.user_resp(thread.recipient, thread.recipient, timeout=15000)
-                print(m.content)
-            except asyncio.TimeoutError:
-                print("timeout")
-                await thread.close(closer=self.bot.modmail_guild.me, message='Closed due to inactivity and not responding to questions')
+            m = await self.wait_for_dm_response(thread.recipient, timeout=timeout)
+            if not m:
+                await thread.close(
+                    closer=self.bot.modmail_guild.me,
+                    message='Closed due to inactivity and not responding to questions.'
+                )
                 return
-            finally:
-                answer = m.content if m.content else "<No Message Content>"
-                answer += "\n"
-                if len(m.attachments) > 0:
-                    for attachment in m.attachments:
-                        answer += f"\n`{attachment.filename}`: {attachment.url}"
-
-                responses[question] = answer
-            print("looping")
-
-        await asyncio.sleep(1)
-        em = discord.Embed(color=self.bot.main_color, timestamp=datetime.utcnow())
-        for k, v in responses.items():
-            em.add_field(name=k, value=v, inline=False)
-        em.set_author(name=m.author.name, icon_url=m.author.avatar_url)
-        message = await thread.channel.send(embed=em)
+
+            answer = m.content.strip() if m.content.strip() else "<No Message Content>"
+            if m.attachments:
+                attachment_details = "\n".join(
+                    f"`{attachment.filename}`: {attachment.url}" for attachment in m.attachments
+                )
+                answer += f"\n{attachment_details}"
+
+            responses[question] = answer
+
+        embed = discord.Embed(color=self.bot.main_color, timestamp=datetime.utcnow())
+        for question, answer in responses.items():
+            embed.add_field(name=question, value=answer, inline=False)
+        embed.set_author(name=thread.recipient.name, icon_url=thread.recipient.avatar.url)
+
+        message = await thread.channel.send(embed=embed)
         await message.pin()
 
-        q_message.content = 'Your appeal will now be reviewed by our moderation team. If you have new information to share about this case, please reply to this message.'
+        # Get the review message from config or use the default
+        review_message = config.get('review_message', 
```

Comment:
I'm personally not big of a fan of having such default messages in here. Maybe a more generic message of "what do you require help with?" would work but this seems too specific to ship by default